dev-libs/LeechCore/LeechCore-2.10.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Direct Memory Access (DMA) Attack Software"
+HOMEPAGE="https://github.com/ufrisk/LeechCore"
+SRC_URI="https://github.com/ufrisk/LeechCore/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="virtual/libusb:*"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	#https://github.com/ufrisk/LeechCore/issues/26
+	emake -C leechcore clean
+	sed -i '/CFLAGS  += -fPIE /d' leechcore/Makefile || die
+
+	eapply_user
+}
+
+src_compile() {
+	emake -C leechcore
+}
+
+src_install(){
+	dolib.so files/leechcore.so
```

Comment:
Huh. This is really all consumers need?

Also, this will.. lack a SONAME, right? Should we install a symlink with it too? Is ABI unstable for this? (May then need to set a subslot & depend on it in consumers with := if linked against).

Reply:
I realized `dev-python/leechcorepyc` ships with a bundled version of `LeechCore` so I decided to drop this ebuild. Thank you anyway for the review. Sometimes it need a hint to see the obvious.

Reply:
No problem. We should ideally fix that / make it use a system copy. Please include a comment about this at least so we don't forget.